extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1087,7 +1093,7 @@ export class AmpA4A extends AMP.BaseElement {
     dev().assert(!!this.element.ownerDocument, 'missing owner document?!');
     this.protectedEmitLifecycleEvent_('renderFriendlyStart');
     // Create and setup friendly iframe.
-    const iframe = /** @type {!HTMLIFrameElement} */(
+    this.iframe = /** @type {!HTMLIFrameElement} */(
```

Comment:
I'll set this.iframe to null in amp-a4a#unlayoutCallback(). Can we handle the correct removal of amp analytics in a separate PR?